### ISSUE TITLE
It says that travisscott.com has a high level of monitor protect.

### ISSUE THREAD

**Issue by CactusJack916** [Dam998/shopify-monitor#13]
can you fix this in any way?

**Comment by Dam998** [comment]
travisscott.com is not a shopify site

**Comment by CactusJack916** [comment]
yeah it is, go check, view page source, ctrl find, shopify, youll see shopify highlighted a bunch

**Comment by ankurghai** [comment]
> yeah it is, go check, view page source, ctrl find, shopify, youll see shopify highlighted a bunch

its shop.travisscott.com. works fine for me.

**Comment by HarshvirP** [comment]
Where would the project folder be for:
Then go inside the project folder and run:

npm install

**Comment by ankurghai** [comment]
@HarshvirP it will be the folder where all the files from this github are stored.

**Comment by HarshvirP** [comment]
@ankurghai Thanks I got that. What I'm confused about is how do I run npm install in the folder? I'm assuming using node? I'm new to programming so any help or direction would be appreciated.

**Comment by HarshvirP** [comment]
@ankurghai nvm got it thanks
